cmake/modules/RootNewMacros.cmake

```diff
@@ -501,7 +501,7 @@ function (ROOT_CXXMODULES_APPEND_TO_MODULEMAP library library_headers)
 
   set(modulemap_entry "module \"${library}\" {")
   # For modules GCocoa and GQuartz we need objc context.
-  if (${library} MATCHES "(libGCocoa|libGQuartz)\\..*")
+  if (${library} MATCHES "(libGCocoa|libGQuartz)")
```

Comment:
Isn't this inconsistent with the change to `${library} MATCHES "(GCocoa|libGQuartz)"` on line 483?